pandas/tests/test_sorting.py

```diff
@@ -188,6 +189,14 @@ def test_nargsort_datetimearray_warning(self):
         with tm.assert_produces_warning(None):
             nargsort(data)
 
+    @pytest.mark.parametrize('data', [
```

Comment:
actually instead of here, can you add the test in  pandas/tests/extension/base/missing.py (and it will be generic with the data_missing fixure)